libshvchainpack/include/shv/chainpack/irpcconnection.h

```diff
@@ -45,10 +45,10 @@ class SHVCHAINPACK_DECL_EXPORT IRpcConnection
 	int callShvMethod(int rq_id, const std::string &shv_path, std::string method, const shv::chainpack::RpcValue &params = shv::chainpack::RpcValue());
 	int callShvMethod(int rq_id, const std::string &shv_path, std::string method, const shv::chainpack::RpcValue &params, const shv::chainpack::RpcValue &user_id);
 
-	int callMethodSubscribe(const std::string &shv_path, std::string method);
-	int callMethodSubscribe(int rq_id, const std::string &shv_path, std::string method);
-	int callMethodUnsubscribe(const std::string &shv_path, std::string method);
-	int callMethodUnsubscribe(int rq_id, const std::string &shv_path, std::string method);
+	int callMethodSubscribe(const std::string &shv_path, std::string method, const std::string& source = "");
```

Comment:
default source should be IMO `get`

Reply:
I don't think it should, by default, `callMethodSubscribe` should subscribe to all the sources. Or I could make the argument mandatory. But this way, older code will be still compatible, and do the same thing.

Reply:
agree